docs/source/suite.rst

```diff
@@ -93,9 +128,52 @@ Rajagopal's full body gait model [https://github.com/opensim-org/opensim-models/
 
 Joints and muscle details can be found in Rajagopal's paper [https://ieeexplore.ieee.org/document/7505900]
 
+The model implementation details can be found on our github repo at `myoLeg <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/leg>`__
```

Comment:
@Vittorio-Caggiano -- 
1. docs are pointing readers to the `myo_sim` repo. Do we want this? 
2. Additionally, they are linking to some weird commit. This should point to latest on `main` 
cc @raku-slyu

Reply:
redirected to the main repo now